### ISSUE TITLE
New Header CSS

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/pomegranate#287]
Should match the library site/orangelight/etc.

**Comment by hackmastera** [comment]
I'll interpret this as follows, please correct if I'm wrong:
1) Update the logo, add site title next to it ala catalog site. 
2) remove banner image area (currently an image of "nassau hall" script) where site title is now.
3) Otherwise update heights, etc, of orange / grey header bar area
4) leave search bar as-is

**Comment by hackmastera** [comment]
this appears to be the relevant orangelight PR https://github.com/pulibrary/orangelight/pull/954/files

**Comment by hackmastera** [comment]
header and footer are pulling from https://github.com/pulibrary/pul_assets; awaiting word from @axamei and co. for next steps.

**Comment by sdellis** [comment]
@hackmastera @axamei and I discussed this, and we think it would be best to wait for us to finish the header and footer patterns in lux and then to add them that way. We have started these, but the Order Manager components were considered higher priority. Let us know if the headers/footers should take priority instead.

**Comment by hackmastera** [comment]
@sdellis I don't think this is higher priority. I guess we'll need webpacker to pull in via lux. I'll make a ticket for that.

**Comment by hackmastera** [comment]
After more discussion re: server-side rendering, it sounds like we should go ahead and update pul_assets for now. It will be some time before lux is ready to take care of this for us.

**Comment by hackmastera** [comment]
okay, holding for lux again!
